runtime/compiler/z/codegen/J9TreeEvaluator.cpp

```diff
@@ -10166,61 +10165,63 @@ genHeapAlloc(TR::Node * node, TR::Instruction *& iCursor, bool isVariableLen, TR
       //   integer arithmetic, checking carry bit is enough to detect it.
       //   For variable length array, we did an up-front check already.
 
-      static char * disableInitClear = feGetEnv("TR_disableInitClear");
-      static char * disableBatchClear = feGetEnv("TR_DisableBatchClear");
-
-      TR::Register * addressReg = NULL, * lengthReg = NULL, * shiftReg = NULL;
-      if (disableBatchClear && disableInitClear==NULL)
+      TR::Register * addressReg = NULL;
+      TR::Register * lengthReg = NULL;
+      TR::Register * shiftReg = NULL;
```

Comment:
Similar to previous comment - We should get this closer to where they are used as much as possible (For example shiftReg is not used except when we are actually generating the zero initialization sequence - Around line 10292 - register allocation and adding it to deps should be done there)

Also the whole block - should be moved after line 10199.

Reply:
done